contracts/token/ERC20/ERC20Burnable.sol

```diff
@@ -8,7 +8,7 @@ import "./ERC20.sol";
  * tokens and those that they have an allowance for, in a way that can be
  * recognized off-chain (via event analysis).
  */
-contract ERC20Burnable is Context, ERC20 {
+abstract contract ERC20Burnable is Context, ERC20 {
```

Comment:
I'm slightly concerned about the user experience around these abstract contracts. Have you played around with using them to see what it feels like, what errors the compiler gives, etc.?

Reply:
Yes, I think error message is reasonable:
```
Compilation errors: 
contracts/mocks/ERC20BurnableMock.sol:5:1: TypeError: Contract "ERC20BurnableMock" should be marked as abstract.
contract ERC20BurnableMock is ERC20, ERC20Burnable {
^ (Relevant source part starts here and spans across multiple lines).
contracts/token/ERC20/ERC20.sol:49:5: Missing implementation:
    constructor (string memory name, string memory symbol, uint8 decimals) public {
    ^ (Relevant source part starts here and spans across multiple lines).
```

It does have an issue issue in that its recommendation is wrong ('contract should be abstract'), but it does point in the direction of the constructor.

Reply:
I think it's a terrible error message but I agree it can be improved on the compiler side.